public/legacy/css/datepicker/css/daterangepicker.css

```
@@ -0,0 +1,520 @@
+
+.daterangepicker .calendar-table th, .daterangepicker .calendar-table td {
+    white-space: nowrap;
+    text-align: center;
+    vertical-align: middle;
+    min-width: 32px;
+    width: var(--day-drp-width);
+    height: var(--day-drp-height);
+    line-height: 20px;
+    font-size: 12px;
+    border-radius: 4px;
+    white-space: nowrap;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

white-space is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)